orttraining/orttraining/core/graph/gradient_builder.cc

```diff
@@ -422,7 +437,7 @@ IMPLEMENT_GRADIENT_BUILDER(GetGemmGradient) {
       HandleBroadcastingDynamic(dY, C, c_shape, IA("dC_reduced"), c_axes, result);
 
       if (has_beta && beta != 1.0f) {
-        NodeDef scale_node = ConstantValueNode(beta, Name("Scale"));
+        NodeDef scale_node = ConstantValueNode(beta, Name("Scale"), I(2).type_proto->tensor_type().elem_type());
```

Comment:
>I(2).type_proto->tensor_type().elem_type() [](start = 68, length = 42)

Consider making a utility function for this ?

Reply:
Just like what we have for IType and OType ?

---
In reply to: [468906830](https://github.com/microsoft/onnxruntime/pull/4717#discussion_r468906830) [](ancestors = 468906830)